src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -24,6 +25,25 @@ public IReadOnlyList<DataRepository> DroppedObservedDataFrom(IDragEvent e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject).ToList();
       }
 
+      public Cache<string, List<DataRepository>> DroppedObservedDataWithFolderPathFrom(IDragEvent e)
```

Comment:
I am having trouble to see where this method is being used?

Reply:
In PK-Sim :D The PK-Sim ChartPresenter has a _observedDataDragDropBinder as private member, and responds to a drag and drop action accordingly. I have finished the PK-Sim branch, but I wanted to first have the Core part review finalized to push that PR too.